{
  "affected": [
    {
      "database_specific": {
        "source": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/arrow/OSV-2020-1228.yaml"
      },
      "ecosystem_specific": {
        "introduced_range": "unknown:ccdc4b076cbd5fb03e1cbc42292def871d2b6035",
        "severity": "HIGH"
      },
      "package": {
        "ecosystem": "OSS-Fuzz",
        "name": "arrow",
        "purl": "pkg:generic/arrow"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "ccdc4b076cbd5fb03e1cbc42292def871d2b6035"
            },
            {
              "fixed": "724da320eaac94003a8f1229d6a820fe8c8c1816"
            }
          ],
          "repo": "https://github.com/apache/arrow.git",
          "type": "GIT"
        }
      ],
      "versions": []
    }
  ],
  "details": "OSS-Fuzz report: https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=20120\n\n```\nCrash type: Heap-use-after-free READ 8\nCrash state:\narrow::internal::CountSetBits\narrow::Array::null_count\narrow::internal::ValidateArray\n```\n",
  "id": "OSV-2020-1228",
  "modified": "2022-04-13T03:04:31.916896Z",
  "published": "2020-07-22T21:49:58.406648Z",
  "references": [
    {
      "type": "REPORT",
      "url": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=20120"
    }
  ],
  "schema_version": "1.7.0",
  "summary": "Heap-use-after-free in arrow::internal::CountSetBits"
}